tests/test_perf_string_reads_and_writes.c

```diff
@@ -0,0 +1,238 @@
+/*
```

Comment:
How do we run the perf tests? Do they run on every TravisCI or we would have to manually run?
Also please share the perf test results in the team.

Reply:
They run on every TravisCI, but the results will not be recorded.